examples/conversational_agent_with_tools.py

```diff
@@ -0,0 +1,75 @@
+import os
+
+from haystack.agents import Tool
+from haystack.agents.base import ToolsManager
+from haystack.agents.conversational import ConversationalAgentWithTools
+from haystack.agents.types import Color
+from haystack.nodes import PromptNode, WebRetriever, PromptTemplate
+from haystack.pipelines import WebQAPipeline
+
+search_key = os.environ.get("SERPERDEV_API_KEY")
+if not search_key:
+    raise ValueError("Please set the SERPERDEV_API_KEY environment variable")
+
+openai_key = os.environ.get("OPENAI_API_KEY")
+if not search_key:
+    raise ValueError("Please set the OPENAI_API_KEY environment variable")
+
+prompt_text = """
+Synthesize a comprehensive answer from the following most relevant paragraphs and the given question.
+Provide a clear and concise answer, no longer than 10-20 words.
+\n\n Paragraphs: {documents} \n\n Question: {query} \n\n Answer:
+"""
+
+prompt_node = PromptNode(
+    "gpt-3.5-turbo",
+    default_prompt_template=PromptTemplate("lfqa", prompt_text=prompt_text),
```

Comment:
`prompt_text` is gone. It's just `prompt` now. https://github.com/deepset-ai/haystack/blob/main/haystack/nodes/prompt/prompt_template.py#L204

Reply:
Yes, excellent, updated with [235d9e6](https://github.com/deepset-ai/haystack/pull/4940/commits/235d9e65785eccab9bb49bb78a0f10b48055207a)